<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadFirmware</class>
 <widget class="QMainWindow" name="uploadFirmware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uploadFirmware</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="group_target">
      <property name="title">
       <string>Target select</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_melding">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_findTarget">
         <property name="text">
          <string>Find target</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_targetIp">
           <property name="text">
            <string>Target Ip:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_ipAdress">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>192.168.2.2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_poort">
           <property name="text">
            <string>Poort:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_poort">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>5555</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_bootlVers">
           <property name="text">
            <string>Bootloader:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_firmVers">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Firmware:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_fileinput">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Open file</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_inputFile">
         <property name="text">
          <string>Selecteer een input file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_openFile">
         <property name="text">
          <string>Open File</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>button_openFile</zorder>
      <zorder>label_inputFile</zorder>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_upload">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Uploaden</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_programming">
           <property name="text">
            <string>Programming:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_programming">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_verifying">
           <property name="text">
            <string>Verifieren:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_verifying">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_upload">
         <property name="text">
          <string>Upload programma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_reset">
         <property name="text">
          <string>Reset Target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_succes">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>19</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
